### ISSUE TITLE
zap_directory resource change events can't be trapped with 'notifies' property.

### ISSUE THREAD

**Issue by jkimalane** [nvwls/zap#44]
Let's assume following:

```
execute 'run_me' do
  action :nothing 
end

zap_directory '/foo.d' do
  notifies :run, 'execute[run_me]', :immediately.
end
```

Now when zap_directory detects unmanaged resource and fires unlink this event is not queued and defined custom action is not executed. It does not matter if the timing is set to 'immediately' or 'delayed'
Expected behaviour would be that in case of change in resource status event is queued and defined action fired.  

This all happens with Chef 13.7.16 and zap cookbook 1.1.0

**Comment by sspans** [comment]
We're having issues with this as well.

Any chance this could be merged?

**Comment by jkimalane** [comment]
To me it seems that someone else should take over maintaining this repo as there seems to be no activity for at least two years.

**Comment by nvwls** [comment]
This should be fixed in v2.1.0
